### PR TITLE
Added contactinformation to all(?) pages

### PR DESCRIPTION
**Pull request by simon-wg**

### CODE REVIEW

**Review by GAsplund — CHANGES_REQUESTED** [review]
Looks good, but I have a few comments.

- The vertical alignment between the text and flags are a bit off:
image
- Check the comments made by Github Actions in the list of changed files

**Comment by simon-wg** [comment]
image
Fixed it now I think, the padding is using rem so it should scale properly with screen size.

**Comment by simon-wg** [comment]
@GAsplund

File: src/components/ContactCard/ContactCard.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          </button>
+        </div>
+        <div className={styles.english}>
+          <button className={styles.button}>
+            <picture className={styles.flag}>
+              <img src="/uk.svg" alt="English flag"/>
+            </picture>
+            <p>English</p>
+          </button>
+        </div>
```
Comment (GAsplund) [inline src/components/ContactCard/ContactCard.tsx]: Make this into an anchor, a button uses the browser's default sans-serif font
```suggestion
        <a className={styles.button}>
          <picture className={styles.flag}>
            <img src="/sweden.svg" alt="Swedish flag"/>
          </picture>
          <p>Svenska</p>
        </a>
        <a className={styles.button}>
          <picture className={styles.flag}>
            <img src="/uk.svg" alt="English flag"/>
          </picture>
          <p>English</p>
        </a>
```

File: src/components/ContactCard/ContactCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    border: none;
+    border-radius: 0.25rem;
+    background-color: transparent;
+    color: var(--text-color);
+    display: flex;
+    p {
+        font-size: 1.25rem;
+        height: 1.25rem;
+    }
+}
```
Comment (GAsplund) [inline src/components/ContactCard/ContactCard.module.scss]: Turn the toggles into flexboxes to align them better
```suggestion
.flag {
    width: 1.25rem;
    height: 1.25rem;
    margin-right: 0.25rem;
}

.link {
    color: var(--link-color);
}

.button {
    cursor: pointer;
    padding: 0.5rem;
    display: flex;
    align-items: center;
    p {
        font-size: 1.25rem;
    }
}

```

File: src/components/ContactCard/ContactCard.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      </div>
+      <Divider />
+      <p>
+        <strong>POSTADRESS</strong>
+      </p>
+      <p>Teknologsektion Informationsteknik</p>
```
Comment (GAsplund) [inline src/components/ContactCard/ContactCard.tsx]: Missing an "en" at the end
```suggestion
      <p>Teknologsektionen Informationsteknik</p>
```

**Review by GAsplund — COMMENTED** [review]
Nicely done! I have a few suggestions to finalize and simplify it a bit. Also, don't forget to run Prettier on the files! IDEs have a tendency to ignore what Prettier says, especially for CSS files with the settings we're using right now.

**Comment by simon-wg** [comment]
Thanks for the suggested changes! That's a much simpler solution than using padding. I've committed them and prettied the scss file now.

**Review by GAsplund — APPROVED** [review]
LGTM!

**Pull request merged**